nsls2forge_utils/dashboard.py

```diff
@@ -5,6 +5,35 @@
 functions had to be re-implemented here.
 """
 from .all_feedstocks import get_all_feedstocks
+from .meta_utils import get_attribute
+
+
+def _extract_org(pkg):
+    try_dev_url = False
+    # get org from home url
+    home_str = get_attribute('about home', pkg, 'nsls-ii-forge')
+    if home_str is None:
+        try_dev_url = True
+    elif 'github.com' in home_str:
+        split_str = home_str.split('/')
+        org = split_str[-2]
+        if org == 'github.com':
+            org = split_str[-1]
+    else:
+        try_dev_url = True
+    # if home url failed try dev_url
+    if try_dev_url:
+        dev_url = get_attribute('about dev_url', pkg, 'nsls-ii-forge')
+        if dev_url is None:
+            return ''
+        elif 'github.com' in dev_url:
+            split_str = dev_url.split('/')
+            org = split_str[-2]
+            if org == 'github.com':
+                org = split_str[-1]
```

Comment:
It's a repeating piece of code. I think to avoid duplication, it's worth to extract it to a helper function (such as `_extract_org_and_repo_from_url` or something like that).